tbxforms/fields.py

```diff
@@ -190,7 +190,7 @@ def clean(self, value):
             days_in_month = calendar.monthrange(year, month)[1]
             if day > days_in_month:
                 error = _(
-                    "Enter a day between 1 to %(dim)s for the month and year "
+                    "Enter a day from 1 to %(dim)s for the month and year "
                     "you entered"
                 ) % {"dim": days_in_month}
```

Comment:
```suggestion
                    "Enter a day from 1 to %(days_in_month)s for the month and year "
                    "you entered"
                ) % {"days_in_month": days_in_month}
```

surely this would make more sense when translating

Reply:
Fixed in [`449f085`](https://github.com/torchbox/tbxforms/pull/29/commits/449f0853079d1aaea40fb41aff78bfdeffff4033)